.gitignore

```diff
@@ -117,3 +117,5 @@ npm-shrinkwrap.json
 .bob/
 
 .hive
+
+cypress
```

Comment:
@dotansimha do you remember why you did it? :D

Reply:
i think it created a weird temp dir